src/aat/resources/features/F-049/F-049.feature

```diff
@@ -0,0 +1,115 @@
+@F-049
+Feature: Start case creation as Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-235
+  Scenario: must start case creation process successfully for correct inputs
+    Given   a user with [a detailed profile in CCD]
+    When    a request is prepared with appropriate values
+    And     the request [provide valid details]
```

Comment:
These 2 lines read almost exactly the same. spec should be more specific or not used.

Reply:
done